flang/test/Driver/lto-flags.f90

```diff
@@ -11,7 +11,7 @@
 
 ! Also check linker plugin opt for Thin LTO
 ! RUN: %flang -### -flto=thin %s 2>&1 | FileCheck %s \
-! RUN:        --check-prefixes=%if system-darwin %{THIN-LTO-ALL%} \
+! RUN:        --check-prefixes=%if system-darwin %{THIN-LTO-ALL%}%if system-aix %{THIN-LTO-ALL,THIN-LTO-LINKER-PLUGIN-AIX%} \
 ! RUN:        %else %{THIN-LTO-ALL,THIN-LTO-LINKER-PLUGIN%}
```

Comment:
```suggestion
! RUN:        --check-prefixes=THIN-LTO-ALL%if system-aix %{,THIN-LTO-LINKER-PLUGIN-AIX%} \
! RUN:          %else %{ %if !system-darwin %{,THIN-LTO-LINKER-PLUGIN%} %}
```

Reply:
Why not write a separate RUN lines for different triples with different prefixes?

Reply:
Indeed, it would be better. It would avoid the nested `%if`s and make it possible to test all targets on the same machine.
But wouldn't it require all tested targets to be built in clang/flang, and tested with a `REQUIRES:` line?

Reply:
I agree that avoiding the nested `%if` is better, I added a new RUN line for powerpc aix to check `-bdbg:thinlto`

Reply:
It seems `-###` doesn't require the tested targets to be built in llvm, because no code is generated, so this is fine.